app/actions/userActions.js

```diff
@@ -24,7 +51,37 @@ export function getUser(id) {
         return userService.getUser(id)
             .then(user => dispatch(getUserSuccess(user)))
             .catch(error => {
-                throw(error);
+                throw (error);
```

Comment:
You're exactly right as usual. This code was copy pasted and formatted only. I plan to do a couple of improvements on a next PR so I will do this change in that one. Thanks for pointing that out. We are not doing anything in this catch.